redbot/core/cog_manager.py

```diff
@@ -360,24 +360,46 @@ async def addpath(self, ctx: commands.Context, *, path: Path):
 
     @commands.command()
     @checks.is_owner()
-    async def removepath(self, ctx: commands.Context, path_number: int):
+    async def removepath(self, ctx: commands.Context, *paths_number: int):
```

Comment:
```suggestion
    async def removepath(self, ctx: commands.Context, *path_numbers: int):
```
I feel that `path_numbers` is better "worded" than `paths_number`. I've changed the docs to use this name in this review